programmer-humour/script.js

```diff
@@ -0,0 +1,19 @@
+const imageComic = document.getElementById("imageComic");
+
+function fetchingImg() {
+  return fetch("https://xkcd.now.sh/?comic=latest")
+    .then((res) => {
+      if (!res.ok) {
+        throw new Error("error");
```

Comment:
Make sure your errors always give some more information about what went wrong. If you just see the text "error", you might be confused about what actually happened